Common/HttpWebRequestWrapperLib/HttpWebRequestWrapper.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+
+namespace HttpWebRequestWrapperLib
+{
+    /// <summary>
+    /// Wrapper of http web requests and responses that encapsulate GET, PUT POST and DELETE requests
+    /// </summary>
+    public class HttpWebRequestWrapper
+    {
+        public HttpWebRequest httpWebRequest { get; set; }
+        public HttpWebResponse httpWebResponse { get; set; }
+
+        public HttpWebRequestWrapper(string contentType = "application/json")
+        {
+            httpWebRequest.ContentType = contentType;
+        }
+        
+        public string Get(string url, Dictionary<string,string> parameters)
+        {
+            if (string.IsNullOrEmpty(url))
+            {
+                return "Url is null or empty";
+            }
+            if (!(parameters is null) && parameters.Count > 0)
+            {
+                url = getUrlWithParams(url, parameters);
+            }
+            httpWebRequest.Method = "GET";
+            HttpWebRequest.Create(url);
+
+            httpWebResponse = (HttpWebResponse)httpWebRequest.GetResponse();
+            using Stream responseStream = httpWebResponse.GetResponseStream();
```

Comment:
var

Reply:
I saw that you already had such a comment and some conversation, but as for me, the type looks obvious. If you don't like var here, tell, which unexpected types could we get here then?

Reply:
what do you mean? stream? it is fixed